packages/api/src/util.ts

```diff
@@ -303,6 +309,14 @@ export function getEnv(): BackendEnv {
     consumerKey: parse('POCKET_CONSUMER_KEY'),
   }
 
+  const subscription = {
```

Comment:
thanks, we should probably do this default thing for more items.

Reply:
yeah, starting to add the default value to the env var